src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,86 @@
+import React from 'react';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (newPage: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const linksAmount = Math.ceil(total / perPage);
+  const linkNumbers = getNumbers(1, linksAmount);
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1 ? 'true' : 'false'}
+          onClick={(event: React.MouseEvent<HTMLAnchorElement>) => {
+            event.preventDefault();
+
+            const newPage = currentPage - 1;
+
+            if (newPage >= 1) {
+              onPageChange(newPage);
+            }
+          }}
+        >
+          «
+        </a>
+      </li>
+
+      {linkNumbers.map(linkNumber => {
+        return (
+          <li
+            className={`page-item ${currentPage === linkNumber ? 'active' : ''}`}
+            key={linkNumber}
+          >
+            <a
+              data-cy="pageLink"
+              className="page-link"
+              href={`#${linkNumber}`}
```

Comment:
Similarly to the previous comment, the `href` attribute here is set to `#` followed by the page number. If these links are not intended for navigation, using a button would be more semantically correct and would not manipulate the URL.